cmd/up/up.go

```diff
@@ -467,10 +467,11 @@ func (up *upContext) getCurrentDeployment(ctx context.Context, autoDeploy bool)
 		return nil, false, err
 	}
 
-	if !autoDeploy {
-		if err := utils.AskIfDeploy(up.Dev.Name, up.Dev.Namespace); err != nil {
-			return nil, false, err
-		}
+	if !autoDeploy && !up.Dev.Autocreate {
```

Comment:
Set `dev.Autocreate` to `true` here:
https://github.com/okteto/okteto/blob/master/cmd/up/up.go#L180
and from there, we only care about `dev.Autocreate`.

Reply:
How will this interact with the CLI param? Will the CLI take precedence?